Find a functionally equivalent code:android.widget.Switch.setThumbResource:COMMENT
Method Modifier: public      
Comment:/**
 * Set the drawable used for the switch "thumb" - the piece that the user
 * can physically touch and drag along the track.
 *
 * @param resId Resource ID of a thumb drawable
 *
 * @attr ref android.R.styleable#Switch_thumb
 */

@@ -1,3 +1,3 @@
 {
-    setThumbDrawable(getContext().getResources().getDrawable(resId));
+    setThumbDrawable(getContext().getDrawable(resId));
 }

